src/App.css

```diff
@@ -62,3 +61,34 @@
 .underline {
     text-decoration: underline;
 }
+
+.line-height-adjustment {
+    text-align: left;
+    margin-inline-start: 0.6em;
+    margin-block-start: 2em;
+
+    #lineHeightInput {
+        padding: 0.4em;
+        margin-inline-start: 0.4em;
+        width: 45px;
+        border-width: 1px;
+        border-radius: 4px;
+        border-color: #808080;
+        text-align: left;
+    }
+}
+
+input[type="number"]::-webkit-inner-spin-button,
+input[type="number"]::-webkit-outer-spin-button {
+    opacity: 1;
+}
+
+.tool-bar {
+    margin-inline-start: 1.25em;
+    margin-block-start: -2.9em;
+    display: flex;
+    background-color: #d9d9d9;
+    height: 8.6vh;
+    width: 60.4vh;
```

Comment:
I could be convinced of this, but it seems a little odd to set width based on viewport height.